### PR TITLE
add blog entry about android-dev

### PR DESCRIPTION
**Pull request by r10s**
@Boehrsi, @florianhaar, all: feel free to comment or to add commits :)

### CODE REVIEW

**Comment by hpk42** [comment]
lgtm -- maybe add a link to a place where new APK are uploaded and maybe mention that 
a) it's a separate app, you can use both "old" and "new" on the same phone for now
b) It's not even "alpha" -- use at your own risk and please feedback to github, not user-support forum. 

?

**Comment by r10s** [comment]
@hpk42 thanks - i will add these points

File: _posts/2018-10-12-android-dev.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+We're currently working on a new Android App for Delta Chat,
+[android-dev](https://github.com/deltachat/deltachat-android-ii/issues).
+
+![Delta-Android-Dev](../assets/blog/20181012-android-dev.jpg)
+
+The new App recreates the UI, however, the core mainly stays unchanged;
+the same core is used for deltachat-desktop and deltachat-ios, btw.
+
+The reason for this rewrite it that the existing ui code,
```
Comment (ralphtheninja) [inline _posts/2018-10-12-android-dev.md]: is that?

File: _posts/2018-10-12-android-dev.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+which is based on another popular messenger,
+uses non-standard programming methods in many cases.
+This makes it hard to maintain and to enhance, especially for new contributors.
+
+We were thinking about this step for months, however, finally, it is started :)
+
+While we'll get a much better source base by this step, 
+we also take this as a chance to improve many things in the new IU, eg.
+
+* Sending text togehter with images
```
Comment (ralphtheninja) [inline _posts/2018-10-12-android-dev.md]: together

File: _posts/2018-10-12-android-dev.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+* Edit images before sending, eg. marking sth.
```
Comment (ralphtheninja) [inline _posts/2018-10-12-android-dev.md]: something

**Review by ralphtheninja — CHANGES_REQUESTED** [review]

**Comment by r10s** [comment]
@ralphtheninja thanks for checking, feel free to merge - we got the okay from @Boehrsi (again, why is your name not in the github menus? i want this too :) via IRC and @florianhaar might already be in the weekend :)

**Pull request merged**